chromadb/api/collection_configuration.py

```diff
@@ -81,7 +81,12 @@ def load_collection_configuration_from_json(
                     embedding_function=None,
                 )
             else:
-                ef = known_embedding_functions[ef_config["name"]]
+                try:
+                    ef = known_embedding_functions[ef_config["name"]]
+                except KeyError:
+                    raise ValueError(
+                        f"embedding function {ef_config['name']} not found. add @register_embedding_function to the class definition."
```

Comment:
could we format these errors a bit nicer? e.g.
```suggestion
                        f"Embedding function {ef_config['name']} not found. Add the @register_embedding_function decorator to the class definition."
```